packages/headless/src/controllers/commerce/core/facets/headless-core-commerce-facet.ts

```diff
@@ -233,6 +231,13 @@ export function buildCoreCommerceFacet<
 
     // Must use a function here to properly support inheritance with `this`.
     toggleSingleExclude: function (selection: ValueRequest) {
+      if (!props.options.toggleExcludeActionCreator) {
+        engine.logger.warn(
```

Comment:
Why would that be of any use for an implementer ? That seems more like a "us" problem, and they could do anything to prevent the warning. 

A warning you can't act upon is aggravating more than anything.

Probably this points to the fact that our code is imprecisely or incorrectly typed if it means we can ever arrive to this point (should that `toggleExcludeActionCreator` be optional in the first place?)